App/CMakeLists.txt

```diff
@@ -59,3 +62,39 @@ install(FILES ../Gui/Resources/Images/natronIcon256_linux.png
     DESTINATION "${CMAKE_INSTALL_DATADIR}/pixmaps")
 install(FILES ../Gui/Resources/Images/natronProjectIcon_linux.png
     DESTINATION "${CMAKE_INSTALL_DATADIR}/pixmaps")
+
+IF(WIN32)
+    SET(CONFIG_FILE ${CMAKE_CURRENT_BINARY_DIR}/install_deps_${CMAKE_BUILD_TYPE}.cmake)
+
+    # Find and install needed DLLs
+    file(GENERATE OUTPUT
+    "${CONFIG_FILE}" CONTENT 
+    [[
+        EXECUTE_PROCESS(COMMAND ${MINGWPATH}/windeployqt.exe ${INSTALL_BINPATH}/Natron.exe)
+        SET(TARGET_APP $<TARGET_FILE:Natron>)
+        FILE(GET_RUNTIME_DEPENDENCIES
+            RESOLVED_DEPENDENCIES_VAR deps_resolved
+            UNRESOLVED_DEPENDENCIES_VAR deps_unresolved
+            LIBRARIES ${TARGET_APP}
```

Comment:
Shouldn't this be EXECUTABLES instead of LIBRARIES since we are getting dependencies for the Natron executable?

Reply:
Perhaps, it's working as it is, but you may be right

Reply:
I don't actually know how are the MINGW libs deployed to make a release, the problem I have now is that the libraries of the OFX plugins need to be installed in the application directory to be found. If they are installed beside the plugin binaries, the linker doesn't find them. A search path issue I guess...